.eslintrc

```diff
@@ -1,21 +1,17 @@
 {
   "extends": "airbnb-base",
-  "env": {
-    "mocha": true,
-    },
   "rules": {
-      "comma-dangle": 0,
-      "max-len": 0,
-      "brace-style" : [2, "stroustrup", {"allowSingleLine" : true}],
-      "no-console": 0,
-      "padded-blocks": 0,
-      "prefer-rest-params": 0,
-      "no-use-before-define": 0,
-      "strict": 0,
-      "no-param-reassign": 0,
-      "no-underscore-dangle": 0
-  },
-  "globals":{
-    "appRequire": true
+    "brace-style" : ["error", "stroustrup", {"allowSingleLine" : true}],
+    "arrow-parens" : ["error", "as-needed"],
```

Comment:
Seems to be more consistent in the code base than the default (which requires parens). It's fixable though, so if you prefer, it's very easy to change